src/lang/de.php

```diff
@@ -35,5 +35,7 @@
     'Add Slot' => 'Slot Hinzufügen',
     'Rooms' => 'Zimmer',
     'Add Room' => 'Zimmer Hinzufügen',
+    'Room' => 'Zimmer',
```

Comment:
Thinking more about it, Raum is probably better than Zimmer. Zimmer is part of an appartment or a house where you live. Raum is more general, and is used for meeting rooms and halls.